patch/config-quanta-ix4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+index 6bd2b13..9292a16 100644
+--- a/debian/build/build_amd64_none_amd64/.config
++++ b/debian/build/build_amd64_none_amd64/.config
+@@ -1723,7 +1723,7 @@ CONFIG_BLK_DEV_LOOP_MIN_COUNT=8
+ CONFIG_BLK_DEV_DRBD=m
+ # CONFIG_DRBD_FAULT_INJECTION is not set
+ CONFIG_BLK_DEV_NBD=m
+-CONFIG_BLK_DEV_NVME=m
++CONFIG_BLK_DEV_NVME=y
```

Comment:
why needs built-in? why cannot use loadable module?

Reply:
![image](https://user-images.githubusercontent.com/19216597/38979281-49a234e8-43ec-11e8-8171-58a225227d9e.png)
hi guohan

our kernel need to put under nvme but not flash, and so that we also have to changed install.sh.
please check above picture, thanks.

Reply:
Hi @lguohan 
Any concern for this ? maybe u got another better way to acheive this goal? thanks.

Reply:
change install.sh is total separate from kernel change.

why do you need to compile nvme as built-in instead of a kernel module. I do not understand the logic here.

Reply:
@lguohan 
IX4 uses PCIe NVME for install kernel image, not PCIe mSATA.
More specifically, the device should be "nvme0n1" but not "mmcblk0".
Any advice to achieve same goal with nvme as kernel module?

Reply:
@lguohan Any advise?